packages/frontend/src/components/Dataset/Dataset.scss

```diff
@@ -1,6 +1,12 @@
 .dataset {
   display: grid;
 
+  .data-link-title{
+    color:red;
+    &:focus-visible{
+      color:aqua;
+    }
+  }
```

Comment:
Don't forget the necessary spaces here after the `:` and before the curly braces, so we can stick to the convention in the rest of the file